src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Establish your current state" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+import cxSegmentWhitelistInvestigation from 'images/cx_screenshot-full_segment-whitelist-investigation.webp'
+
+import cxQfDashboard from 'images/cx_screenshot-full_qf-dashboard.webp'
+
+import cxQfKpiProgress from 'images/cx_diagram_qf_kpi_progress.webp'
+
+The first step in any improvement experience is to establish your baselines so that you have numbers to measure your KPIs against and can track improvement over time, and improving your customer experience is no exception to this rule. Once you check to make sure everything you need is reporting correctly to our platform, you can validate your information and use our quality foundation dashboard to measure your customer experience.
+
+<Steps>
+    <Step>
+### Review instrumented pages
```

Comment:
Should there be a line break between these step tags and the header?

Reply:
Sure should!